src/App.jsx

```diff
@@ -1,107 +1,41 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+import { PostList } from './components/PostList';
+
+export function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+export function getUserByEmail(email) {
+  return usersFromServer.find(user => user.email === email);
+}
+
+export function getCommentsByPostID(id) {
+  return comments.filter(comment => comment.postId === id);
+}
+
+// робимо копію масиву коментарів з commentsFromServer та додаємо туди властивість author(як об'єкт)
+export const comments = commentsFromServer.map(comment => ({
+  ...comment,
+  author: getUserByEmail(comment.email),
+}));
```

Comment:
Please clarify, where you use this comments array? If it's nowhere then please delete it